lib/src/modules/common/utils/open_link.dart

```diff
@@ -17,19 +17,19 @@ Future<void> openLink(BuildContext context, String url) async {
 
 Future<void> openPath(BuildContext context, String url) async {
   if (Platform.isWindows) {
-    await Process.start(
```

Comment:
We probably should remove these strings from translation.

Reply:
Fixed in main